spring-cloud-stream-binders/spring-cloud-stream-binder-redis/pom.xml

```diff
@@ -44,6 +44,11 @@
 		</dependency>
 		<dependency>
 			<groupId>org.springframework.cloud</groupId>
+			<artifactId>spring-cloud-cloudfoundry-connector</artifactId>
```

Comment:
In general I'd refrain from "factoring out" dependencies. I think it makes sense for s-c-s-modules up to the point where every pulled up dep is 100% sure to be used/useful.
In the particular case you describe, the cf stuff may well be useless to future binders, so I'd -1